### DISCUSSION TITLE
Official OWL Driver Board now available!

### DISCUSSION THREAD

**Discussion by geezacoleman** [geezacoleman/OpenWeedLocator#108] [Announcements]
The official OWL Raspberry Pi hat is now available for purchase from [Paradar](https://paradar.co.uk/products/owl-driver-board-raspberry-pi-automotive-relay-hat).

My brother designed the board to incorporate:
1. Power supply
2. Relay control
3. Solenoid connections
4. PWM (in future iterations)
5. Fuses
6. Pi protection

All in the one Raspberry Pi hat. It allows a Raspberry Pi to drive 4x 2A outputs in any 12V or 24V system (working voltage of 7 - 26V). It is able to tolerate high levels of electrical noise, reverse polarity, short circuit conditions and over voltage.

![image](https://github.com/geezacoleman/OpenWeedLocator/assets/51358498/c7226000-9bb3-44e6-84e0-f15b0bbee142)

As with the rest of the OWL, [all hardware designs are open-source](https://github.com/geezacoleman/owl-driver-board) and available if you want to have your own built. Alternatively you can purchase them from the Paradar website as ready to use for £39. I do have a discount code too if you're interested.

This should drastically simplify assembly, while improving relay response times, compactness and durability. Any/all feedback welcome on the design too if you think features should be added/changed.

**Comment by benboughton1** [comment]
This is great work. I’ve ordered one to test.

**Reply by benboughton1** [reply]
I’m curious now this changes the bill of materials to test owl on a bench. I really just need pi, camera, this board, power source and leds?
Plus the wire & connectors. I’m not familiar with them.

**Reply by geezacoleman** [reply]
Nice! Will be good to get your feedback on the design and how it can be improved/changed.

Yeh, the bill of materials is much shorter now. The instructions need a bit of an overhaul, as do the 3D model files, though I think it would be good to keep a non-owl driver board option too. 

The minimum requirements to run a bench test would be:
1. Raspberry Pi + SD card
2. Camera + lens
3. OWL driver board
4. LEDs - either for visualising the detections and/or for status
5. Buzzer
6. Wiring - power in/connection to each of the solenoids/LEDs etc. Connectors are already on the driver board, so unless you're running a panel mount connector then you'd be able to get away without it.
7. Standoffs for the Pi and the driver board
8. Enclosure - not entirely necessary for a bench test
9. Power source too + any connections required
10. Optional - Google Coral for running green on green algorithms


Much more straightforward!

A few mages of the bench setup I used for testing. Should have everything above, except the buzzer.
| Image 1 | Image 2 | Image 3 |
| --- | --- | --- |
| ![Image 1](https://github.com/geezacoleman/OpenWeedLocator/assets/51358498/cbc645a1-e277-4661-919a-ec79732a3d75) | ![Image 2](https://github.com/geezacoleman/OpenWeedLocator/assets/51358498/010451a8-81e1-4785-8457-279678b2d23c) | ![Image 3](https://github.com/geezacoleman/OpenWeedLocator/assets/51358498/772a9f18-ae54-4d59-9c0e-e81fb134e5e5) |

**Reply by benboughton1** [reply]
That’s helpful thanks.
